perf_tests/coldstart/client.mjs

```diff
@@ -0,0 +1,69 @@
+import { performance } from 'node:perf_hooks'
+import v8 from 'v8'
+
+import { OneClient } from '../../packages/nodejs_host/dist/index.js'
+
+async function flushMetrics() {
+  let id = ''
+  for (const entry of performance.getEntries()) {
+    if (entry.entryType === 'mark' && Array.isArray(entry.detail)) {
+      id = entry.detail.join('-')
+    } else if (entry.entryType === 'measure') {
+      console.log(`${entry.startTime},${id},${entry.name},${entry.duration}`)
+    }
+  }
+
+  performance.clearMarks()
+  performance.clearMeasures()
+
+  const heap = v8.getHeapStatistics()
+  console.log(`${performance.now()},${id},__heap_size,${heap.total_heap_size}`)
+  // process.stderr.write(JSON.stringify(heap, undefined, 2))
+}
+
+async function outerIteration(outerIteration, innerIterations) {
+  performance.mark('init:start', { detail: [outerIteration] })
+  const client = new OneClient({
+    assetsPath: '.',
+    superfaceApiUrl: false,
+    onBeforeExitHook: false
+  })
+  const profile = await client.getProfile('coldstart')
+  performance.measure('init', 'init:start')
+
+  for (let i = 0; i < innerIterations; i += 1) {
+    if (i % 100 === 0) {
+      // periodically clean up metrics because otherwise we run out of memory
+      await client.sendMetricsToSuperface()
+    }
+
+    performance.mark('perform:start', { detail: [outerIteration, i] })
+    let result = await profile
+      .getUseCase('ColdStart')
+      .perform(
+        { i: i % 256 },
```

Comment:
Is the module here, because you figured out there is difference with bigger number?